### PR TITLE
fix: minor fix to deadlocked taskrun text

### PR DESCRIPTION
**Pull request by gabemontero**
Signed-off-by: Gabe Montero <gmontero@redhat.com>

rh-pre-commit.version: 2.3.1
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

**Comment by gabemontero** [comment]
@amisstea - since this is just text on a panel, I do not things we need to bother App SRE with this, correct?

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 1]
```diff
@@ -1342,7 +1342,7 @@ data:
                     "type": "prometheus",
                     "uid": "${datasource}"
                 },
-                "description": "The number of validated TaskRuns not yet processed by the TaskRuns Controller.",
+                "description": "The number of validated TaskRuns not yet processed by the TaskRuns Controller.  Negative numbers mean no TaskRuns Controller deadlocks despite Kubernests throttling.",
```
Comment (amisstea) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1345]:
```suggestion
                "description": "The number of validated TaskRuns not yet processed by the TaskRuns Controller.  Negative numbers mean no TaskRuns Controller deadlocks despite Kubernestes throttling.",
```

Comment (gabemontero) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: thanks updated

**Review by amisstea — APPROVED** [review]

**Comment by gabemontero** [comment]
I've rebased this PR @amisstea

**Pull request merged**